R/binomialtestbayesian.R

```diff
@@ -110,124 +110,126 @@ BinomialTestBayesian <- function(jaspResults, dataset = NULL, options, ...) {
   } else {
     bfTitle <- paste0("BF", bfTitleSpec[["null"]], bfTitleSpec[["other"]])
   }
-  
+
   binomTable$addColumnInfo(name = "case",       title = "",                    type = "string", combine = TRUE)
   binomTable$addColumnInfo(name = "level",      title = gettext("Level"),      type = "string")
   binomTable$addColumnInfo(name = "counts",     title = gettext("Counts"),     type = "integer")
   binomTable$addColumnInfo(name = "total",      title = gettext("Total"),      type = "integer")
   binomTable$addColumnInfo(name = "proportion", title = gettext("Proportion"), type = "number")
   binomTable$addColumnInfo(name = bfType,       title = bfTitle,               type = "number")
-  
+
   if (options$hypothesis == "lessThanTestValue")
     note <- gettextf("For all tests, the alternative hypothesis specifies that the proportion is less than %s.", options$testValueUnparsed)
   else if (options$hypothesis == "greaterThanTestValue")
     note <- gettextf("For all tests, the alternative hypothesis specifies that the proportion is greater than %s.", options$testValueUnparsed)
   else
     note <- gettextf("Proportions tested against value: %s.", options$testValueUnparsed)
 
+  note <- gettextf("%1$s Shape of the prior distribution under the alternative hypothesis is specified by Beta(%2$s, %3$s).", note, options[["priorAUnparsed"]], options[["priorBUnparsed"]])
```

Comment:
maybe?
```suggestion
  note <- gettextf("%1$s The shape of the prior distribution under the alternative hypothesis is specified by Beta(%2$s, %3$s).", note, options[["priorAUnparsed"]], options[["priorBUnparsed"]])
```
Assuming this leads to sentences like:

```
"Proportions tested against value: 0.5. The shape of the prior distribution under the alternative hypothesis is specified by Beta(1.0, 1.0)."
```

Reply:
Thanks, I was worried that the dutch would be incorrect, turns out I English cannot write also 😄